examples/example_tabular_classification.py

```diff
@@ -43,7 +44,9 @@ def get_data_to_train() -> typing.Tuple[typing.Any, typing.Any, typing.Any, typi
         X=X_train, Y=y_train,
         X_test=X_test, Y_test=y_test)
 
-    api = TabularClassificationTask(delete_tmp_folder_after_terminate=False)
+    api = TabularClassificationTask(delete_tmp_folder_after_terminate=False,
+                                    temporary_directory='./tmp/example_tabular_classification_tmp',
```

Comment:
Can you change back to not using a provided directory? For the examples it make sense to try the most common behaviour, which is not providing... (During unit testing we test providing a path, so this is like the only place we test not having this temporay_directory=<> flag)

Reply:
yeah sure